docs/features/roll-editor.md

```diff
@@ -30,10 +30,64 @@ The metadata for a weapon will show what damage type it is, is it a primary, sec
 
 ## Weapon Stats
 
+A weapon's stats can placed into two categories; its basic stats and its advanced stats. The basic stats are what are shown on the weapon info screen in the game. For most weapons they are as follows: impact, range, stability, handling, reload speed, aim assistance, zoom, and airborne effectivenes. Some weapon types have stats that are unique to that type of weapon. The following are the unique stats for that given weapon type. Swords have the charge rate, guard resistance, guard effciency, and guard endurance stats. Rocket launchers and grendade launchers have the blast radius and velocity stats. Combat bows have accuracy and draw time stats. Fusion and linear fusion rifles have the charge time stat. Glaives have the shield duration stat.
+
+A weapon's advanced stats aren't shown on the weapon info screen directly or are not made entirely clear what the number means in terms of gameplay. There are things such as the damage profile, flinch resist, and reserves. These advanced stats are the result of combining two or more of the basic stats to give you a tangible value to see in the game. The range stat alone doesn't determine when your damage fall off occurs. Range and zoom together are what define the weapon's falloff distance among other things.
+
+This part of the document seeks to answer most questions about various stats, however, it should be noted that Destiny 2 is a very complex game. More in-depth explainations of these stats and scenarios will be expanded upon in their own document. 
+
 ### Basic
 
+The basic stats are there to give you a rough idea of how a weapon feels and operates. They should only be compared with weapons of the same type as stats such as impact are relative to the base impact number of a weapon type. The aim of the basic stat section is to describe the stats in relation to other weapons of the same type as well as give small examples as to how they work in game.
+
+#### Impact
+
+Impact is the damage stat on most weapons. This number can be a little misleading, however, as it can only be used to reference damage within the same weapon type. E.g. the hand cannon "Targeted Redaction" has an impact stat value of 92 and does 80 crit damage in the crucible. Compare that to the combat bow "Tyranny of Heaven" which has an impact stat of 68 and does 137 crit damage in the crucible. For the most part, impact values are the same across a given frame for a weapon type. It mainly comes into play with fusion rifles, including linear fusions, and swords. Those weapon types can augement their impact stat via certain perk selections.
```

Comment:
Hm... this might be slightly misleading? Impact is ultimately purely visual, even if a stat increase to it such as in the case of Fusion Rifles, etc is accompanied with some change to damage (like Matador has higher impact than other Precision Frame Shotguns, but doesn't actually do more damage). It's a good visual indicator across a weapon family of their relative damage, but it bears no mathematical value

Reply:
I agree with you and this stat gave me the most trouble if im being honest... toiled on it for about two days. the problem is that bequest genuinely does more damage than other adaptive frame swords. I really wanted to write 'this stat means nothing 99% of cases' but idk. if you want to change it so it reads that it's purely visual then that is fine with me. I was also mainly looking out for new players that might not know impact is visual and ask the hand cannon/bow type question.